release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,6 +6,18 @@ uid: DCP_change_log
 
 The DataMiner Cloud Platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 13 January 2023 - Enhancement - Auditing of protocol and script deployments [ID_35392]
+
+The *Audit* page of the DCP Admin app will now also contain records for protocol and script deployments, while previously it only showed records for DxM artifact deployments.
```

Comment:
Not sure if this is **new**, we must check if this wasn't audited already. This was more like a bug fix I think, Koen?

Reply:
We fixed it via a bug fix on our sprint indeed but i documented it as New in RN because our original implementation only targeted dxm deployments. Did not consider dead letters in service bus at the time when people started using other deployments. Based on this, would you prefer bug fix or new feature?

Reply:
discussed with Jordy and agreed to keep as bug.